etl/steps/data/garden/ahdi/2023-09-08/augmented_hdi.meta.yml

```diff
@@ -0,0 +1,294 @@
+common:
+  entity_annotations: "Western offshoots (AHDI): United States, Canada, Australia and New Zealand"
+dataset:
+  title: Augmented Human Development Index (AHDI)
+  update_period_days: 1095
+tables:
+  augmented_hdi:
+    variables:
+      ahdi:
+        title: Augmented Human Development Index (AHDI)
+        unit: ""
+        short_unit: ""
+        description_short: |
+          The Augmented Human Development Index (AHDI) measures human wellbeing with a multidimensional index composed of four different dimensions: healthy life, knowledge, freedom, and wellbeing dimensions other than education and health.
```

Comment:
You should consider `description_short` as a continuation of `title`, therefore, you don't need to repeat the definition of AHDI. You could simply start as "Measures human wellbeing...".